pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -497,6 +500,22 @@ func (l *LibvirtDomainManager) SetGuestTime(vmi *v1.VirtualMachineInstance) erro
 	return nil
 }
 
+func (l *LibvirtDomainManager) GetLoopbackAddress() string {
+	//DEBUG
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("DBGDBG Recovered in f", r)
+		}
+	}()
```

Comment:
I suppose you will remove this later? ;)

Reply:
Hehe ye, trying to debug how to fix the travis failure and the mock